### PR TITLE
build: release: v1.26.0-rc1

### PR DESCRIPTION
**Pull request by rjan90**
## Proposed Changes
- Update and clean up the changelog, highlighting features, etc.

## Checklist

Before you mark the PR ready for review, please make sure that:

- [x] Commits have a clear commit message.
- [x] PR title is in the form of of `<PR type>: <area>: <change being made>`
  - example: ` fix: mempool: Introduce a cache for valid signatures`
  - `PR type`: fix, feat, build, chore, ci, docs, perf, refactor, revert, style, test
  - `area`, e.g. api, chain, state, market, mempool, multisig, networking, paych, proving, sealing, wallet, deps
- [ ] If the PR affects users (e.g., new feature, bug fix, system requirements change), update the CHANGELOG.md and add details to the UNRELEASED section.
- [ ] New features have usage guidelines and / or documentation updates in
  - [ ] [Lotus Documentation](https://lotus.filecoin.io)
  - [ ] [Discussion Tutorials](https://github.com/filecoin-project/lotus/discussions/categories/tutorials)
- [ ] Tests exist for new functionality or change in behavior
- [ ] CI is green

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,69 @@
 
 ## Improvements
 
-### Tracing API
+# v1.26.0-rc1 / 2024-02-28
+
+This is the first release candidate of the upcoming MANDATORY Lotus v1.26.0 release, which will deliver the Filecoin network version 22, codenamed Dragon 🐉.
+
+**This release candidate sets the calibration network to upgrade at epoch 1427974, which is 2024-03-11T14:00:00Z**
+
```
Comment (jennijuju) [inline CHANGELOG.md:12]:
```suggestion
This release does NOT set the mainnet upgrade epoch yet, in which will be updated in the final release.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -47,10 +109,28 @@ Replace the `CodeCid` field in the message trace (added in 1.23.4) with an `Invo
...
+3. Selfdestruct operations are not included in the trace.
+4. EVM smart contract "create" events always specify `0xfe` as the "code" for newly created EVM smart contracts.
+
+Additionally, Filecoin is not Ethereum no matter how much we try to provide API/tooling compatibility. This API attempts to translate Filecoin semantics into Ethereum semantics as accurately as possible, but it's hardly the best source of data unless you _need_ Filecoin to look like an Ethereum compatible chain. If you're trying to build a new integration with Filecoin, please use the native `StateCompute` method instead.
+
+[handlefilecoinmethod]: https://fips.filecoin.io/FIPS/fip-0054.html#handlefilecoinmethod-general-handler-for-method-numbers--1024
+
 ### Lotus API
-2 new methods have benn added to the Lotus API called `GetAllAllocations` and `GetAllClaims` which lists all the available allocations and claims available in the actor state.
+2 new methods have been added to the Lotus API called `GetAllAllocations` and `GetAllClaims` which lists all the available allocations and claims available in the actor state.
```
Comment (jennijuju) [inline CHANGELOG.md:133]: 4 more no? Missing the actor events APIs

Comment (rjan90) [reply CHANGELOG.md:133]: Yes, I will need to rebase this PR first.

File: CHANGELOG.md [hunk 3]
```diff
@@ -27,7 +27,70 @@ See [FIP-0045](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-004
...
+init		          bafk2bzaced5sq72oemz6qwi6yssxwlos2g54zfprslrx5qfhhx2vlgsbvdpcs
+multisig	        bafk2bzacedbgei6jkx36fwdgvoohce4aghvpohqdhoco7p4thszgssms7olv2
+paymentchannel    bafk2bzaceasmgmfsi4mjanxlowsub65fmevhzky4toeqbtw4kp6tmu4kxjpgq
+placeholder       bafk2bzacedfvut2myeleyq67fljcrw4kkmn5pb5dpyozovj7jpoez5irnc3ro
+reward            bafk2bzacedjyp6ll5ez27dfgldjj4tntxfvyp4pa5zkk7s5uhipzqjyx2gmuc
+storagemarket     bafk2bzaceabolct6qdnefwcrtati2us3sxtxfghyqk6aamfhl6byyefmtssqi
+storageminer      bafk2bzaceckzw3v7wqliyggvjvihz4wywchnnsie4frfvkm3fm5znb64mofri
+storagepower      bafk2bzacea7t4wynzjajl442mpdqbnh3wusjusqtnzgpvefvweh4n2tgzgqhu
+system            bafk2bzacedjnrb5glewazsxpcx6rwiuhl4kwrfcqolyprn6rrjtlzmthlhdq6
+verifiedregistry  bafk2bzacednskl3bykz5qpo54z2j2p4q44t5of4ktd6vs6ymmg2zebsbxazkm
```
Comment (masih) [inline CHANGELOG.md:48]:
```suggestion
Actor             CID  
account           bafk2bzaceb3j36ri5y5mfklgp5emlvrms6g4733ss2j3l7jismrxq6ng3tcc6
cron              bafk2bzaceaz6rocamdxehgpwcbku6wlapwpgzyyvkrploj66mlqptsulf52bs
datacap           bafk2bzacea22nv5g3yngpxvonqfj4r2nkfk64y6yw2malicm7odk77x7zuads
eam               bafk2bzaceatqtjzj7623i426noaslouvluhz6e3md3vvquqzku5qj3532uaxg
ethaccount        bafk2bzacean3hs7ga5csw6g3uu7watxfnqv5uvxviebn3ba6vg4sagwdur5pu
evm               bafk2bzacec5ibmbtzuzjgwjmksm2n6zfq3gkicxqywwu7tsscqgdzajpfctxk
init              bafk2bzaced5sq72oemz6qwi6yssxwlos2g54zfprslrx5qfhhx2vlgsbvdpcs
multisig          bafk2bzacedbgei6jkx36fwdgvoohce4aghvpohqdhoco7p4thszgssms7olv2
paymentchannel    bafk2bzaceasmgmfsi4mjanxlowsub65fmevhzky4toeqbtw4kp6tmu4kxjpgq
placeholder       bafk2bzacedfvut2myeleyq67fljcrw4kkmn5pb5dpyozovj7jpoez5irnc3ro
reward            bafk2bzacedjyp6ll5ez27dfgldjj4tntxfvyp4pa5zkk7s5uhipzqjyx2gmuc
storagemarket     bafk2bzaceabolct6qdnefwcrtati2us3sxtxfghyqk6aamfhl6byyefmtssqi
storageminer      bafk2bzaceckzw3v7wqliyggvjvihz4wywchnnsie4frfvkm3fm5znb64mofri
storagepower      bafk2bzacea7t4wynzjajl442mpdqbnh3wusjusqtnzgpvefvweh4n2tgzgqhu
system            bafk2bzacedjnrb5glewazsxpcx6rwiuhl4kwrfcqolyprn6rrjtlzmthlhdq6
verifiedregistry  bafk2bzacednskl3bykz5qpo54z2j2p4q44t5of4ktd6vs6ymmg2zebsbxazkm
```

**Review by masih — COMMENTED** [review]
CIDs look good to me 👍

**Review by masih — APPROVED** [review]

File: build/params_calibnet.go [hunk 4]
```diff
@@ -89,8 +89,8 @@ const UpgradeWatermelonFixHeight = 1070494
 // 2023-11-21T13:00:00Z
 const UpgradeWatermelonFix2Height = 1108174
 
-// 2024-03-06T14:00:00Z
-const UpgradeDragonHeight = 1413574
+// 2024-03-11T14:00:00Z
+const UpgradeDragonHeight = 1427974
```
Comment (arajasek) [inline build/params_calibnet.go:93]: Epoch LGTM.

**Review by arajasek — APPROVED** [review]

File: CHANGELOG.md [hunk 5]
```diff
@@ -70,8 +112,44 @@ Replace the `CodeCid` field in the message trace (added in 1.23.4) with an `Invo
...
+1. The Geth APIs are not implemented, only the OpenEthereum (Erigon, etc.) APIs.
+2. Block rewards are not (yet) included in the trace.
+3. Selfdestruct operations are not included in the trace.
+4. EVM smart contract "create" events always specify `0xfe` as the "code" for newly created EVM smart contracts.
+
+Additionally, Filecoin is not Ethereum no matter how much we try to provide API/tooling compatibility. This API attempts to translate Filecoin semantics into Ethereum semantics as accurately as possible, but it's hardly the best source of data unless you _need_ Filecoin to look like an Ethereum compatible chain. If you're trying to build a new integration with Filecoin, please use the native `StateCompute` method instead.
+
+[handlefilecoinmethod]: https://fips.filecoin.io/FIPS/fip-0054.html#handlefilecoinmethod-general-handler-for-method-numbers--1024
+
+### GetActorEvents and SubscribeActorEvents
```
Comment (jennijuju) [inline CHANGELOG.md:135]:
```suggestion
// TODO: add links to the documentation when #11615 lands

### GetActorEvents and SubscribeActorEvents
```

**Review by jennijuju — APPROVED** [review]

**Pull request merged**